### PR TITLE
Add Shoko episodeID Copy Button / Trim Descriptions

### PR DESCRIPTION
**Pull request by natyusha**
firefox_2024-10-02_11-13-58

### CODE REVIEW

File: src/components/Collection/Episode/EpisodeDetails.tsx [hunk 1]
```diff
@@ -13,6 +13,10 @@ const getDuration = (duration: string) => {
   return `${intMinutes} minutes`;
 };
 
+const handleCopyToClipboard = (id: string) => {
+  copyToClipboard(id, 'Shoko episodeID').catch(console.error);
```
Comment (harshithmohan) [inline src/components/Collection/Episode/EpisodeDetails.tsx:17]: maybe name it `Shoko Episode ID`

File: src/components/Collection/Episode/EpisodeFiles.tsx [hunk 2]
```diff
@@ -82,7 +82,7 @@ const EpisodeFiles = ({ anidbSeriesId, episodeFiles, episodeId, seriesId }: Prop
     });
 
   const handleCopyToClipboard = (id: string) => {
-    copyToClipboard(id, 'ShokoID').catch(console.error);
+    copyToClipboard(id, 'Shoko fileID').catch(console.error);
```
Comment (harshithmohan) [inline src/components/Collection/Episode/EpisodeFiles.tsx:85]: same as above `Shoko File ID`

**Review by harshithmohan — CHANGES_REQUESTED** [review]

File: src/components/Collection/Episode/EpisodeDetails.tsx (hunk 1 above)
Comment (natyusha) [reply src/components/Collection/Episode/EpisodeDetails.tsx]: That does look better in the toast (changed the icon of the button too)

File: src/components/Collection/Episode/EpisodeFiles.tsx (hunk 2 above)
Comment (natyusha) [reply src/components/Collection/Episode/EpisodeFiles.tsx]: 👍

**Comment by natyusha** [comment]
I also added a minor change to description text and renamed the PR since I noticed some filtered descriptions with leading/trailing newlines.

**Pull request merged**